tests/release/e2e-test-default-bundle.go

```diff
@@ -91,7 +91,13 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-
 			componentDetected = compDetected
 		}
 
-		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+		_, err = fw.AsKubeAdmin.HasController.CreateApplication(applicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		component, err = fw.AsKubeAdmin.HasController.CreateComponent(componentDetected.ComponentStub, devNamespace, "", "", applicationNameDefault, false, map[string]string{})
```

Comment:
Use CDQ and from cdq create the component

Reply:
thanks @flacatus , 
since this PR is for different issue, is it fine by you if I solve this comment in a separate PR that I'll create ?

Reply:
@flacatus , created a dedicated ticket for this https://issues.redhat.com/browse/RHTAPREL-610